### ISSUE TITLE
Use cypress-axe for accessibility tests

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#3289]
## User Story or Problem Statement

As a developer, I need an up-to-date accessibility testing framework to ensure that accessibility issues may be detected and easily fixed.

This is a potential alternative to implementing #3274

## Acceptance Criteria
* [ ] cypress-axe is implemented and giving equivalent results to axe + phantomjs
* [ ] update server dependencies
   * [ ] local
   * [ ] CI
* [ ] turn off ignore-failures in yaml tests
* [ ] Tests are not run on BRD environment

**Comment by ElijahLynn** [comment]
Like this direction!
